docs-2.0/nebula-operator/8.custom-cluster-configurations/8.5.enable-ssl.md

```diff
@@ -14,236 +14,598 @@ In the NebulaGraph environment running in Kubernetes, mutual TLS (mTLS) is used
 
         In the cluster created using Operator, the client and server use the same CA root certificate by default. 
 
-## Create a TLS-type Secret
+## Encryption policies
 
-In a K8s cluster, you can create Secrets to store sensitive information, such as passwords, OAuth tokens, and TLS certificates. In NebulaGraph, you can create a Secret to store TLS certificates and private keys. When creating a Secret, the type `tls` should be specified. A `tls` Secret is used to store TLS certificates.
+NebulaGraph provides three encryption policies for mTLS:
 
-For example, to create a Secret for storing server certificates and private keys:
+- Encryption of data transmission between the client and the Graph service.
```

Comment:
The existing ssl related config is a mess. @dutor is redesigning the configs. So Operator and the doc requires update within these few days as well.